kapitan/inventory/inv_reclass.py

```diff
@@ -44,6 +43,8 @@ def render_targets(self, targets: list = None, ignore_class_notfound: bool = Fal
             for target_name, rendered_target in rendered_inventory["nodes"].items():
                 self.targets[target_name].parameters = rendered_target["parameters"]
                 self.targets[target_name].classes = rendered_target["classes"]
+                self.targets[target_name].applications = rendered_target["applications"]
```

Comment:
Could we not also copy all key/values in `rendered_target`? why are we hardcoding specific keys?

Reply:
I think I see why, it is required for the keys to be known. ok

Reply:
I have no hard opinion here. I didn't find a helper for dataclasses which generates a dataclass from a dict (although I didn't search for very long), so I ended up just copying the fields.